ur-packages/hex-string/__tests__/hexstring.test.ts

```diff
@@ -0,0 +1,104 @@
+import { HexString } from "../src";
+import { URRegistryDecoder } from "@keystonehq/bc-ur-registry";
+
+describe("hex-string", () => {
+    const hex = "babecafe8badf00d";
+    const buff = Buffer.from(hex, "hex");
+
+    const expectedCBOR = "D9010748BABECAFE8BADF00D".toLowerCase(); // Should we?
```

Comment:
So this is the cbor data represented as a hex? I would rename this, because it sounds like the cbor Buffer you expect.

Reply:
It is the hex representation of the CBOR Buffer.
It is easier to compare across js engines when its string.

Reply:
indeed, but I would name it as such